control/tests/iosys_test.py

```diff
@@ -51,7 +54,7 @@ class TSys:
     def test_linear_iosys(self, tsys):
         # Create an input/output system from the linear system
         linsys = tsys.siso_linsys
-        iosys = ios.LinearIOSystem(linsys)
+        iosys = ios.LinearIOSystem(linsys).copy()
```

Comment:
is it obvious why this `.copy` is necessary?

Reply:
Definitely not obvious, but this gets rid of a warning later on since the `interconnect` function generates a warning if a system appears more than once in an interconnected system, since then the name of system is not unique.  (It's also possible to turn the warning off with `warn_dupicate=False`).